playground/terraform/infrastructure/cloudbuild-manual-setup/README.md

```diff
@@ -0,0 +1,142 @@
+  Ensure that the account has at least the following privileges:
+    - Service Account Admin
+    - Storage Admin
+    - Service Usage Admin
+    - Cloud Build Editor
+    - Security Admin
+    - Service Account User
+- [gcloud CLI](https://cloud.google.com/sdk/docs/install-sdk)
+- An existing GCP Bucket to save Terraform state - `state_bucket`
+- DNS name for your Playground deployment instance
+- [Terraform](https://www.terraform.io/)
+- [Apache Beam GitHub](https://github.com/apache/beam) repository cloned locally
+
+## 1. Setup the Google Cloud Build  for your GCP project
+
+The `playground/terraform/infrastructure/cloudbuild-manual-setup/01.setup` provisions dependencies required to setup Cloud Build for Playground:
+- Required API services
+- Cloud Build service account
+- IAM permissions for Cloud Build service account
+- GCP Storage Bucket for Cloud Build logs (Note: [name must be globally unique](https://cloud.google.com/storage/docs/buckets#:~:text=Bucket%20names%20reside%20in%20a,responds%20with%20an%20error%20message.))
+
+#### To execute the module:
+
+1. Create a folder `playground/terraform/environment/{environment_name}` (further referred to as `environment_name`) to define a new environment for Playground.
+2. Create `terraform.tfvars` file in the newly created `environment_name` folder.
+3. Fill the `terraform.tfvars` configuration file as follows:
+
+```
+project_id                    = "project_id"                # Your Project ID
+region                        = "us-central1"               # Your GCP region name where resources will be provisioned
+location                      = "us-central1-b"             # Select the deployment location from available in the specified region
+cloudbuild_service_account_id = "cloudbuild_sa_name"        # Name of SA to be used by Cloud Build to deploy Playground
+cloudbuild_logs_bucket_name   = "logs_bucket_name"          # Assign a globally unique name of GCP bucket for Cloud Build logs
+github_repository_name        = "beam"                      # The name of the GitHub repo. Example: In https://github.com/example/foo is 'foo'
+github_repository_owner       = "owner_name"                # Owner of the GitHub repo. Example: In https://github.com/example/foo is 'example'.
+github_repository_branch      = "branch_name"               # The name of the GitHub repo branch
+
+network_name                  = "network_name"              # GCP VPC Network Name for Playground deployment
+gke_name                      = "playground-backend"        # Playground GKE Cluster name
+state_bucket                  = "bucket_name"               # GCS bucket name for Beam Playground temp files
+bucket_examples_name          = "bucket_name-example"       # GCS bucket name for Playground examples storage
+```
+
+5. Push the `environment_name` folder to your repository.
```

Comment:
Should it be pushed to 'github_repository_branch' branch?

Reply:
Yes, right